### ISSUE TITLE
Support for iOS 9+ in the AppDelegate

### ISSUE THREAD

**Issue by AnthonyOliveri** [ibm-bluemix-mobile-services/bms-clientsdk-swift-security-facebookauthentication#19]
The sample app and `FacebookAuthenticationManager` are using an old AppDelegate method, `application(_:open:sourceApplication:annotation)`. This method was deprecated in iOS 9. We need to update the documentation, sample app, and the `onOpenURL` method in `FacebookAuthenticationManager` to use the new AppDelegate method, `application(_:open:options)`.

**Comment by odedBetzalel** [comment]
Fixed it

**Comment by AnthonyOliveri** [comment]
Thanks! I confirmed that it works.

I want to point out 2 things though:
1. The new `onOpenURL` method calls the old `FBSDKApplicationDelegate` method, but there is a new one it could use instead that has the same parameters (_:open:options). Both `FBSDKApplicationDelegate` methods seem to work though, so nothing is broken by this.
1. This was released as a patch version, but since a new method was added to the API, a minor version (2.1.0) would have been more appropriate. 

Since you already released a new version and it works, I don't see any urgent need to release a new minor version, unless you really want to.
